.changeset/witty-jokes-hammer.md

```diff
@@ -0,0 +1,10 @@
+---
+'@axis-backstage/plugin-jira-dashboard-backend': major
+'@axis-backstage/plugin-jira-dashboard-common': minor
+---
+
+Introduced TypeScript type definitions SearchJiraResponse and JiraQueryResults to represent Jira search responses and pagination details.
```

Comment:
Since this changes the "searchJira" call in a breaking way please update this with code examples how it has changed and how to update any code that is using it.